Mathlib/Algebra/Module/EquivNatInt.lean

```diff
@@ -0,0 +1,55 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro, Kevin Buzzard, Yury Kudryashov, Frédéric Dupuis,
+  Heather Macbeth
+-/
+import Mathlib.Algebra.Module.Equiv
+import Mathlib.Algebra.Module.Hom
```

Comment:
Could you move the entire content of this new file to `Algebra.Module.Equiv` instead?